recipes/fairlearn/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.4.0" %}
+
+package:
+  name: fairlearn
+  version: {{ version }}
+
+source:
+  url: https://github.com/fairlearn/fairlearn/archive/v{{ version }}.tar.gz
+  sha256: 38ee80303ede1cddf119cbc0bf08f337978103f0e90badef97c64087fd4613f2
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  number: 0
+
+  # If the installation is complex, or different between Unix and Windows, use
+  # separate bld.bat and build.sh files instead of this key. By default, the
+  # package will be built for the Python versions supported by conda-forge and
+  # for all major OSs. Add the line "skip: True # [py<35]" (for example) to
+  # limit to Python 3.5 and newer, or "skip: True # [not win]" to limit to
+  # Windows.
+
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

Comment:
This seems like a noarch package to me. Is anything version specific?
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

Reply:
I'd agree, but linter disagreed: https://github.com/conda-forge/staged-recipes/pull/10518#issuecomment-570553673